code/game/mecha/mecha.dm

```diff
@@ -1892,12 +1897,25 @@
 		operation_req_access += topic_filter.getNum("add_req_access")
 		output_access_dialog(topic_filter.getObj("id_card"),topic_filter.getMob("user"))
 		return
+	if(href_list["add_all_req_access"] && add_req_access && topic_filter.getObj("id_card"))
+		if(!in_range(src, usr))
+			return
+		for(var/a in topic_filter.getObj("id_card").access)
```

Comment:
iirc this works the same as a trust operator, so you should separate it out into different lines

Reply:
I'm not sure what that means or how I would do as you ask.
```
var/card = getobj,
var/accessed = card.access
for var/a in accesses
```
?

Reply:
Yes, that's fine